api-inference-community/requirements.txt

```diff
@@ -1,5 +1,5 @@
 starlette==0.14.2
-numpy==1.20.2
+numpy>=1.18.0
```

Comment:
That's also required because flair requires numpy <1.20.

Reply:
Should this then be "numpy>=1.18.0,<1.20.0"?

Reply:
No, because `api-inference-community` doesn't want to limit the numpy requirements, here we just enable old versions (the numpy usage is very limited so we shouldn't encounter any issues. `pip` will solve the dependency.

Reply:
Alright, makes sense!